modin/core/storage_formats/base/query_compiler.py

```diff
@@ -2077,7 +2077,7 @@ def drop(self, index=None, columns=None):
     # UDF (apply and agg) methods
     # There is a wide range of behaviors that are supported, so a lot of the
     # logic can get a bit convoluted.
-    def apply(self, func, axis, *args, **kwargs):
+    def apply(self, func, axis, raw=False, result_type=None, *args, **kwargs):
```

Comment:
explicitly dedicate special parameters that [are passed here from front-end](https://github.com/modin-project/modin/blob/fea20642ac30104fe2bc4cd7b616e756f2cfac23/modin/pandas/base.py#L862-L863)

Reply:
Why don't we explicitly pass these parameters to pandas QC?

Reply:
We don't need to dedicate these params at the pandas QC, they naturally go through `*args` and `**kwargs` to the native `pandas.DataFrame.apply`. I've dedicated these params at the base QC just because it declares an interface for all QCs and so all of the parameters have to be explicitly present.